tests/errors/cppexc_non_extern.pyx

```diff
@@ -14,9 +14,9 @@ cdef test_func2(self) except +:
     pass
 
 _ERRORS = """
-9:16: Only extern functions can throw C++ exceptions.
+9:15: Only extern functions can throw C++ exceptions.
```

Comment:
I'd say, the column here should be either 6 (the function name) or 23 (the except clause) or 30 (the `+` sign of the except clause).

Reply:
I'll change it to point at the name (5 in 0-indexing) since that's easiest to get a position for